Sources/PexipInfinityClient/Conference/ConferenceService.swift

```diff
@@ -196,7 +196,22 @@ struct DefaultConferenceService: ConferenceService {
 
             switch response.statusCode {
             case 200:
-                return try parse200(from: data)
+                let token = try parse200(from: data)
+                if token.version.versionId < "29" {
+                    logger?.warn(
+                        """
+
+                        WARNING: Infinity v\(token.version.versionId) support.
+
+                        We offer only limited support for Infinity versions prior to v29.
+                        You may experience problems with sending and receiving presentation.
+                        Use `presentationInMain` to mix presentation with main video feed.
+
+                        This will be a fatar error in Q4 2023.
+                        """
+                    )
+                }
+                return token
```

Comment:
I wonder if the better place to put it is somewhere around conference creation rather in the response parsing? I think it's safe to use just the client API bit for white-glove use cases.

Reply:
Makes sense, will put it in `InfinityClientFactory`